tests/FSharp.Test.Utilities/Compiler.fs

```diff
@@ -683,6 +686,7 @@ module rec Compiler =
             match csSource.OutputType with
             | Exe -> OutputKind.ConsoleApplication, "exe"
             | Library -> OutputKind.DynamicallyLinkedLibrary, "dll"
+            | Module -> OutputKind.NetModule, "mod"
```

Comment:
I've never knew it was a thing...

Reply:
In reality modules are output with a .dll extension.

Reply:
but this isn't the extension just a string that identifies it.